FHEM/14_Hideki.pm

```diff
@@ -422,22 +421,6 @@ sub wind {
 	return ($windchill,$windspeed,$windgust,$winddir,$winddirdeg,$winddirtext);
 }
 
-sub
```

Comment:
@elektron-bbs 

Ich bin überrascht, dass das Modul fehlerfrei ohne AttrFn funktioniert. Ich nehme stark an, Du hast das getestet?

Reply:
Da in der Sub sowieso nichts anderes behandelt wurde als IODev, kann sie auch weg.

```
sub
Hideki_Attr(@)
{
  my @a = @_;

  # Make possible to use the same code for different logical devices when they
  # are received through different physical devices.
  return if($a[0] ne "set" || $a[2] ne "IODev");
  my $hash = $defs{$a[1]};
  my $iohash = $defs{$a[3]};
  my $cde = $hash->{CODE};
  delete($modules{Hideki}{defptr}{$cde});
  $modules{Hideki}{defptr}{$iohash->{NAME} . "." . $cde} = $hash;
  return undef;
}

```

Reply:
Dass es sonst nichts macht habe ich gesehen. Ich konnte auf die Schneller auch nichts finden, ob dass Funktion vorhanden sein darf.
Scheint also ohne zu funktionieren :)

Reply:
Ich hatte bisher auch noch nichts gelesen, das die Sub vorhanden sein **muss**.